lib/aytests/cli.rb

```diff
@@ -40,11 +51,16 @@ def test(*files)
             provider: options["provider"] || ENV["AYTESTS_PROVIDER"] || :libvirt,
             headless: options[:headless] || ENV["AYTESTS_HEADLESS"] == "true"
           )
-          runner.run
+          # When a test fails, a non-zero return code will be returned
+          results[test_file] = runner.run ? :passed : :failed
         else
+          results[test_file] = :missing
           $stderr.puts "File #{test_file} does not exist"
```

Comment:
Sort of log file should be better from my POV. Searching for this reports on stderr is not comfortable (I know I can redirect output into a file but who cares when starting the test ;-)